static/app/components/nav/issueViews/issueViewNavItemContent.tsx

```diff
@@ -68,14 +138,129 @@ export function IssueViewNavItemContent({
         onChange={() => {}}
         setIsEditing={setIsEditing}
       />
+      {view.unsavedChanges && (
+        <Tooltip
+          title={constructUnsavedTooltipTitle(view.unsavedChanges)}
+          position="top"
+          skipWrapper
+        >
+          <UnsavedChangesIndicator
+            role="presentation"
+            data-test-id="unsaved-changes-indicator"
+            isActive={isActive}
+          />
+        </Tooltip>
+      )}
     </StyledSecondaryNavReordableItem>
   );
 }
 
+const READABLE_PARAM_MAPPING = {
+  query: t('query'),
+  querySort: t('sort'),
+  projects: t('projects'),
+  environments: t('environments'),
+  timeFilters: t('time range'),
+};
+
+const constructUnsavedTooltipTitle = (unsavedChanges: Partial<IssueViewPFParams>) => {
+  const changedParams = Object.keys(unsavedChanges)
+    .filter(k => unsavedChanges[k as keyof IssueViewPFParams] !== undefined)
+    .map(k => READABLE_PARAM_MAPPING[k as keyof IssueViewPFParams]);
+
+  return (
+    <Fragment>
+      {t(`This view's `)}
```

Comment:
This will result in a bad translation because the translators will not see the entire context of the sentence, just the start and the end separately